src/test/resources/TestNG.xml

```diff
@@ -3,10 +3,12 @@
 
 <suite name="TestNG">
 
-    <test name="Example Test" preserve-order="false">
+    <test name="Example Test" preserve-order="true">
         <classes>
             <class name="LoginTests"/>
             <class name="BaseTest"/>
+            <class name="Homework17"/>
+            <class name="Homework16"/>
         </classes>
     </test>
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/223036785-f201d4cb-185a-48d4-900b-42b53e3025c6.png)

**Hint** 

`public void setLogEmail(@Optional String mail, @Optional String pass)`